pkgs/top-level/all-packages.nix

```diff
@@ -1240,7 +1240,7 @@ in
 
   fedora-backgrounds = callPackage ../data/misc/fedora-backgrounds { };
 
-  fedora-coreos-config-transpiler = callPackage ../development/tools/fedora-coreos-config-transpiler { };
+  fedora-coreos-config-transpiler = throw "fedora-coreos-config-transpiler has been renamed to 'butane'.";
```

Comment:
Please move this to aliases.nix.

Reply:
Ah, so that is where it goes, thanks for pointing that out! Moved.